### PR TITLE
Added contributors

### PR DESCRIPTION
**Pull request by ecurrencyhodler**
Added people involved with bringing ltcd up to date with btcd.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,3 +139,9 @@ verify the signature perform the following:
 ## License
 
 ltcd is licensed under the [copyfree](http://copyfree.org) ISC License.
+
+## Contributers
```
Comment (jlamur) [inline README.md]: typo here

Comment (ecurrencyhodler) [reply README.md]: Thanks!

**Review by thrasher- — APPROVED** [review]

**Comment by thrasher-** [comment]
👍 thanks!

**Comment by losh11** [comment]
CLOSING: we will include this in a `CONTRIBUTIONS.md` file.

**Pull request closed without merging**